apps/builder/app/routes/rest.patch.ts

```diff
@@ -64,6 +70,13 @@ export const action = async ({ request }: ActionArgs) => {
     throw Error(`Build ${buildId} not found`);
   }
 
+  const serverVersion = build.version;
+  if (clientVersion !== serverVersion) {
+    return {
+      status: "version_mismatched",
```

Comment:
nit: why underscore separated?

Reply:
btw maybe we should start organizing these statuses as consts, since we likely will add more in the future? and it would help prevent typos as this is not being typechecked